### DISCUSSION TITLE
開発環境時におけるportの割り振りについて

### DISCUSSION THREAD

**Discussion by ghost** [su-its/ams-project#9] [Ideas]
@equal-l2 @h-takeyeah

nodejsは比較的3000番をエンドポイントにすることが多いのですが、

nuxtは比較的容易に変えることが出来るので、バックエンドではなく、フロントを変更しようと思います。

基本的な骨子としては

```
API: 3000
MariaDB:3306
フロント:3001
```

で開発していこうと思いますが如何でしょうか。

**Comment by h-takeyeah** [comment]
いいと思います．その場合，CORS対策をバックエンド側に施さないといけないのでしょうか．もし必要なら多分できますのでおっしゃってください．

**Reply by ghost** [reply]
chromeの場合ほぼ間違いなく必要だと思います。

また、すべてのブラウザでテスティングしないと意味がなくなってしまいます。
CORSしなくても良いと最初から考えるのではなく、やるべきことの中には入れておくべきかと...

**Comment by ghost** [comment]
@h-takeyeah ^
それと、これはあくまで現在の段階でのportの割り当てです。実稼働した場合、portを変えないとdev環境を構築することが出来ないので
4月になったら考えなければならない、ということは頭の片隅においておいてください。
